org.eclipse.winery.frontends/app/tosca-management/src/app/instance/instance.service.ts

```diff
@@ -64,12 +64,15 @@ export class InstanceService {
                     subMenu.push(SubMenuItems.parameters);
                 }
                 if (!this.configurationService.isYaml()) {
-                    subMenu.push(SubMenuItems.documentation, SubMenuItems.plans, SubMenuItems.selfServicePortal, SubMenuItems.researchObject,
+                    subMenu.push(SubMenuItems.documentation, SubMenuItems.plans, SubMenuItems.selfServicePortal,
                         SubMenuItems.boundaryDefinitions, SubMenuItems.tags, SubMenuItems.constraintChecking,
                         SubMenuItems.xml);
                     if (this.configurationService.configuration.features.nfv) {
                         subMenu.push(SubMenuItems.threatModeling);
                     }
+                    if (this.configurationService.configuration.features.researchObject) {
+                        subMenu.push((SubMenuItems.researchObject));
+                    }
```

Comment:
Whats the point? I don't think the research object stuff has anything to do with this.